web/public/templates/errors/404_no_headers.dust

```diff
@@ -1,5 +1,20 @@
 <div class="container">
-  <div>
-    {@i18n key="Page not found"/}
+  <div class="container">
+    <div class="col-xs-12 col-sm-10 col-sm-offset-1 text-center">
+      <div class="ninja-container">
+        <div class="ninja"></div>
+      </div>
+      <div class="page-not-found-caption">
+        <h4>{@i18n key="Oops! This is not the page you are looking for."/}</h4>
+        <h5>{@i18n key="Maybe you'd like to check out some of the following fun sections of our site:"/}</h5>
+        <a href="http://zen.coderdojo.com">Find your local Dojo</a>
```

Comment:
All of the strings here in each of the links - "Find your local Dojo", etc. will have to be added to messages.po and wrapped in the i18n tags also! Sorry didn't spot it before :)